src/stats/show_stats.py

```diff
@@ -125,6 +161,23 @@ def make_percentage_graph(stat_id, options, answers, title, graph_size=None):
     plt.clf()
 
 
+def make_time_line_plot(stat_id, data_points, data_values, title, graph_size=None):
```

Comment:
`make_timeline_plot`

Reply:
Renamed in [3214a02](https://github.com/StreetOfCode/discord-bot/pull/43/commits/3214a02c2989735a84a4664407f1318286ceb138).